porthtml.html

```diff
@@ -78,15 +78,152 @@ <h6 class="LET"><a href="#">LET’S CONNECT</a></h6>
          
           
         </ul>
+        </div>
+        
         <img src="abc.svg" alt="" width="20" height="20">
 
         <img src="rec.svg" alt="" width="20" height="20">
 
+      
+
+    <div id="modal" class="modal modal--active">
+         
+        
+      <div>
+      <img class="whole-card" src="card2.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+      <img class="whole-card" src="card1.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+        <img class="whole-card" src="card3.svg" alt="Work preview for project Uber Navigation">
+        </div>
+        <br>
+        <div>
+          <img class="whole-card" src="card4.svg" alt="Work preview for project Uber Navigation">
+          </div>
```

Comment:
- [ ] Instead of using images to display website, You have to write HTML code for this work section.
- The container of the section is a `section` tag.
- The main container of the projects has a `display:grid`.
- For each project
 The content of the project is aligned with flexbox
 The project title (e.g., "Tonic") use a header tag `hx`
 The project has an image, that could use a `img` or use create it as a background
 The paragraph (e.g., "A daily section ...") is a `p` tag
 The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)
![vereev](https://user-images.githubusercontent.com/26767383/153653242-c5d93368-49df-452e-be97-acd36dbc797e.PNG)

* The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`